.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: dart
 dart:
-  - stable
+  - 1.19.1
```

Comment:
There's an issue with coverage on more recent versions that I'll need to get fixed in dart_dev.